# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/bluez/hotfix-pause.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- bluez-5.60/profiles/audio/media.c.vanilla	2021-08-02 15:43:17.943771480 +0200
+++ bluez-5.60/profiles/audio/media.c	2021-08-02 15:44:05.512773934 +0200
@@ -1281,7 +1281,7 @@
 	return media_player_send(mp, "Stop");
 }
 
-static bool pause(void *user_data)
+static bool blu_pause(void *user_data)
 {
 	struct media_player *mp = user_data;
 
@@ -1331,7 +1331,7 @@
 	.set_volume = set_volume,
 	.play = play,
 	.stop = stop,
-	.pause = pause,
+	.pause = blu_pause,
 	.next = next,
 	.previous = previous,
 };
